lib/code_corps/map_utils.ex

```diff
@@ -8,18 +8,18 @@ defmodule CodeCorps.MapUtils do
   def keys_to_string(map), do: stringify_keys(map)
 
   # Goes through a list and stringifies keys of any map member
-  def stringify_keys(nil), do: nil
-  def stringify_keys(%DateTime{} = val), do: val
-  def stringify_keys(map = %{}) do
+  defp stringify_keys(nil), do: nil
+  defp stringify_keys(%DateTime{} = val), do: val
```

Comment:
That said, we do _not_ need L11 (`stringify_keys(nil)`) because that _does_ get picked up by `stringify_keys(val), do: val`.